backends/p4fmt/attach.h

```diff
@@ -0,0 +1,45 @@
+#ifndef BACKENDS_P4FMT_ATTACH_H_
+#define BACKENDS_P4FMT_ATTACH_H_
+
+#include <filesystem>
+#include <unordered_map>
+#include <vector>
+
+#include "ir/visitor.h"
+#include "lib/source_file.h"
+
+namespace P4::P4Fmt {
+
+class Attach : public Transform {
```

Comment:
Why this is a `Transform`? It seems you are only populating maps and not changing anything at all.

Reply:
`Transform` Pass was used when comments were attached directly to IR nodes. Now that we’ve switched to using an side map, there's indeed no need for the this. I’ll revert it to `Inspector`.

Reply:
This is taken care of now.